server/stores.go

```diff
@@ -169,6 +171,19 @@ func (s *Store) Dashboards(ctx context.Context) chronograf.DashboardsStore {
 	return &noop.DashboardsStore{}
 }
 
+// OrganizationConfig returns a noop.OrganizationConfigStore if the context has no organization specified
+// and an organization.OrganizationConfigStore otherwise.
+func (s *Store) OrganizationConfig(ctx context.Context) chronograf.OrganizationConfigStore {
+	if isServer := hasServerContext(ctx); isServer {
```

Comment:
Do we need this case?

Reply:
Good catch -- seems like there's no need to guard against this since these are all organization-specific. We'll plan to remove it

Reply:
removed.